docs/modules/ROOT/pages/reference/billing.adoc

```diff
@@ -0,0 +1,66 @@
+= Billing
+
```

Comment:
Also rename the file to `billing-naming.adoc` or something

Reply:
Done, but I feel like the example query doesn't fit anymore, if we make the page only about the naming.

Reply:
I know, but I'm fine with leaving it for now until we have a better place.